### ISSUE TITLE
Wiki 3.3 Tools and Materials

### ISSUE THREAD

**Issue by alidursen** [swe-ms-boun/2018fall-swe574-g1#71]
I think it is time that we start at least listing what are we going to use.
Which DB solution, SW framework etc? Thus we can research them if we need to. So far it only lists Git as source control (obviously) and Axure RP for mockups. Those are useful of course, but not really essential.

**Comment by vedatakses** [comment]
You are right @alidursen ! I'll be updating this page with the tools and technologies that will be used in the project.

**Comment by taneresme** [comment]
Exactly, I will be also updating the page according to the technologies we are going to use on the frontend side.

**Comment by tugcanolgun** [comment]
Updated for UI dependencies

**Comment by vedatakses** [comment]
Updated with Docker and PostgreSQL parts.
